tools/collect-workflow.sh

```diff
@@ -42,6 +42,10 @@ if [[ -z $WORKFLOW ]]; then
     echo "Must specify -w"
     exit 1
 fi
+if [[ -z $WORKDIR ]]; then
+    echo "Must specify -d"
+    exit 1
+fi
```

Comment:
if this is missing, could you use the `-w` param? I've been using the workflow name as the directory name and it'd be nice to have `-d` default to that.

Reply:
That sounds good.